crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -1599,13 +1599,15 @@ where
             EnginePruneEvent::Started(tip_block_number) => {
                 trace!(target: "consensus::engine", %tip_block_number, "Pruner started");
                 self.metrics.pruner_runs.increment(1);
+                self.sync_state_updater.update_sync_state(SyncState::Syncing);
```

Comment:
I'd like a note here why we're doing this, basically, because prune+fcu/engine are mutually exclusive so we're not making progress while we're pruning